microsoft-outlook/latest/config/index.md

```diff
@@ -633,7 +641,21 @@ See examples of how to use these search settings below.
 
     If the above example is applied to the **Custom simple search** configuration, the Outlook Integration uses the search criteria mentioned in the `<target>` element only to find search results.
 
-6. Click **Apply** to save your changes and restart Microsoft Outlook.
+6. To apply search criteria using an aspect, use this example:
+
+    ```xml
+    <match type="aspect" pattern="cm:versionable">
```

Comment:
The aspect cm:versionable does not have the properties cm:title and cm:description, they are part of the cm:titled aspect

Reply:
It's only an example mapping. 
If i.e. a folder has the cm:versionable aspect, please search for title and description when using a search inside this folder.

A use case could be:
The customer has a folder with contracts. This contract folder has the custom aspect ca:contracts assigned.
If a user uses a search request within this folder, the admin could configure that the search uses the property ca:contractId.

He could do that before as well, but needed to specify the folder path instead.

Reply:
Thanks for the clarification @mhoffmann-itnovum. I've now agreed with @gravitonian that this change can be merged into the interim branch.